src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -160,60 +160,64 @@ private AdsDataProvider() {
 
   @DataProvider
   public static Object[][] skin() {
-    final String wikiName = "project43";
-    final String pageName = "SyntheticTests/Skin";
+    final String noSkinWikiName = "adtest";
```

Comment:
I don't think so because not in every case we need no-skin-wiki but by accident, it's the same wiki